xchange-gdax/src/main/java/org/knowm/xchange/gdax/service/GDAXBaseService.java

```diff
@@ -15,7 +15,7 @@
 
 public class GDAXBaseService extends BaseExchangeService implements BaseService {
 
-  protected final GDAX gdax;
+  private final GDAX gdax;
```

Comment:
Doing it this way diverges from the way all other modules are implemented in XChange, so I'd rather not see this here and the related gdax() method.

Reply:
OK, since we don't need the 404 handling code, I don't need this.  It was only here to allow test mocking.